sdk/search/azure-search-documents/README.md

```diff
@@ -180,6 +432,28 @@ When you interact with Azure Cognitive Search using this Java client library, er
 to the same HTTP status codes returned for [REST API][rest_api] requests. For example, if you try to retrieve a document 
 that doesn't exist in your index, a `404` error is returned, indicating `Not Found`.
 
+### Handling Search Error Response
+
+Any Search API operation that fails will throw a
+[`HttpResponseException`][HttpResponseException] with
+helpful [`Status codes`][status_codes].  Many of these errors are recoverable.
+
+<!-- embedme ./src/samples/java/com/azure/search/documents/ReadmeSamples.java#L110-L118 -->
+```Java
+try {
+    Iterable<SearchResult> results = searchClient.search("hotel");
+} catch (HttpResponseException ex) {
+    // The exception contains the HTTP status code and the detailed message
+    // returned from the search service
+    HttpResponse response = ex.getResponse();
+    System.out.println("Status Code: " + response.getStatusCode());
+    System.out.println("Message: " + ex.getMessage());
+}
+```
+
+You can also easily [enable console logging][logging] if you want to dig
```

Comment:
Don't see a section for advanced/complete samples. 
We don't have that for Search?